.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+  source = serenata_toolbox
+  omit =
```

Comment:
I don't think we need this `omit` (and the following line).

Once we setup `source = serenata_toolbox` the only think `coverage` will take into account is the `serenata_toobox` module (aka the directory `serenata_toolbox` in the root of the repo).

Besides that each user has a different name their for virtualenv (`~/.virtualenvs/` for those using the default of `virtualenvwrapper`, `.venv`, `venv`, `env`, `.env` etc…).

Any way, would you mind testing to check what reslts the `.coveragerc` without the `omit` gives us?